src/main/java/org/prebid/server/bidder/epsilon/EpsilonBidder.java

```diff
@@ -260,11 +260,12 @@ private Bid updateBidWithId(Bid bid) {
     }
 
     private static BidType getType(String impId, List<Imp> imps) {
-        for (Imp imp : imps) {
-            if (imp.getId().equals(impId)) {
-                return imp.getVideo() != null ? BidType.video : BidType.banner;
-            }
-        }
-        return BidType.banner;
+        return imps.stream()
+                .filter(imp -> impId.equals(imp.getId()))
+                .findFirst()
+                .map(imp -> imp.getAudio() != null ? BidType.audio
+                        : (imp.getVideo() != null ? BidType.video
+                        : BidType.banner))
+                .orElse(BidType.banner);
```

Comment:
I'd probably go this way, because it's closer to the existing bidders style
```java
    private static BidType getType(String impId, List<Imp> imps) {
        for (Imp imp : imps) {
            if (imp.getId().equals(impId)) {
                if (imp.getVideo() != null) {
                    return BidType.video;
                } else if (imp.getAudio() != null) {
                    return BidType.audio;
                } else {
                    return BidType.banner;
                }
            }
        }
        return BidType.banner;
    }
```

Reply:
done